src/picongpu/include/fields/FieldB.hpp

```diff
@@ -51,7 +52,7 @@ namespace picongpu
     {
     public:
         typedef float3_X ValueType;
-        typedef typename promoteType<float_64, ValueType>::type UnitValueType;
```

Comment:
note for myself: would only be required if the types here would be templates for `FieldB`.